README.md

```diff
@@ -77,13 +77,24 @@ return [
 
 ## Usage
 
+``` php
+php artisan make:notification ExpoNotification
+```
+App\Notifications\ExpoNotification
 ``` php
 use NotificationChannels\ExpoPushNotifications\ExpoChannel;
 use NotificationChannels\ExpoPushNotifications\ExpoMessage;
 use Illuminate\Notifications\Notification;
 
-class AccountApproved extends Notification
+class ExpoNotification extends Notification
 {
+    private $message;
+    
+    public function __construct($message)
```

Comment:
The default notification created from the command `php artisan make:notification` does not come with the above constructor, I would prefer to remove the `$message` param from the constructor... in other words I prefer the to keep the old example. Thanks :)

Reply:
This will help dev how to pass parameters to notification message using dependency injection design pattern

Reply:
Yes, but that's out of the scope of the README.. the purpose of the document is to illustrate how to use the package :)

Reply:
This is a point of view, you can remove it. but I prefer to add it because it shows a full example of usage.

Reply:
can I push new pull or not?